pkg/store/config.go

```diff
@@ -0,0 +1,71 @@
+// Copyright (c) The Thanos Authors.
+// Licensed under the Apache License 2.0.
+
+package store
+
+import (
+	"fmt"
+
+	"gopkg.in/yaml.v2"
+
+	"github.com/prometheus/prometheus/discovery/file"
+)
+
+type Config struct {
+	Name            string          `yaml:"name"`
+	TlsConfig       *TlsConfig      `yaml:"tls_config"`
+	EndpointsConfig EndpointsConfig `yaml:",inline"`
+}
+
+type TlsConfig struct {
```

Comment:
ditto (albeit obvious), but maybe describe the tls config of what is this actually.